src/GafferSceneUI/TransformTool.cpp

```diff
@@ -900,8 +910,14 @@ void TransformTool::updateSelection() const
 		m_selection.begin(), m_selection.end(),
 		[]( const Selection &a, const Selection &b )
 		{
-			auto ta = a.editTarget(), tb = b.editTarget();
-			return ta != a.editScope() && tb != b.editScope() && ta == tb;
+			const auto ta = editTargetOrNull( a );
+			const auto tb = editTargetOrNull( b );
+			return
+				ta && tb &&
+				ta != a.editScope() &&
+				tb != b.editScope() &&
+				ta == tb
```

Comment:
Sanity checking thought process - we don't wan't to de-dupe `ta == tb == null` as they may have different warnings?

Reply:
Correct